Sources/Widgets/Representations/LineHandleRepresentation/index.js

```diff
@@ -96,6 +96,7 @@ function defaultValues(initialValues) {
     _pipeline: {
       glyph: vtkCylinderSource.newInstance({
         resolution: initialValues.glyphResolution ?? 4,
+        initAngle: initialValues.glyphAngle ?? 0,
```

Comment:
shouldn't it be PI/4 by default ?

Reply:
In my opinion it is the responsibility of the reslice cursor to set the right angle for its use
The LineHandleRepresentation doesn't have a good reason to have a rotation by default

Reply:
The good reason is that the default resolution is 4.

If it was not 4, I agree it does not make sense to have an angle != 0.

But in this case it is explicit that we want a rectangular prism that is aligned with orthogonal axes.

No ?

Reply:
The alignment with the axes is a requirement of the reslice cursor
Is it also a default requirement for the other widgets using this representation?

Reply:
I think it is a good default for ALL the lines, not just RCW.

Reply:
Ok no problem, it is fixed!